### ISSUE TITLE
How to disable the "graphic" pronunciation in alternative texts?

### ISSUE THREAD

**Issue by rafaelbald** [nvaccess/nvda#7129]
Hello,
Sorry for my english, I'm using the google translator.

I work in a public university in Brazil and I am part of a project that seeks to understand and create a standardization in the elaboration of digital archives accessible to scientific publications and other magazines of the institution. Our audience is aimed at those with partial or total vision difficulties.

In short, we want to establish standards for the creation of accessible PDFs, which are read by NVDA, in a coherent and inherent way to the visual media.

During the test run, we had difficulty with how the screen reader ends up repeating the term "graphic" many times each time it encounters an alternative text function.

How to disable the "Graphic" quote at every encounter with the alternative text function in reading images from PDF files?

Greatful,

**Comment by jage9** [comment]
I for some reason thought this was under document formatting but don't 
see it there. Perhaps it would be an option to consider? We can turn off 
just about everything else in browse mode.

**Comment by rafaelbald** [comment]
So, I'm new to the program. But I noticed some older update notes, where they described this option to disable it. But in this current version I found neither the option nor the update notes. For our proposal, it would be interesting to have the possibility to disable and enable to put in the profiles. It is useful for content that is not thought of with accessibility methods.
Momentarily, can not I disable it by some command in the python terminal?
